docs/controlling-fixtures.md.bak

```diff
@@ -0,0 +1,443 @@
+---
```

Comment:
This file should not be included

Reply:
Oops, didn't spot that sneaking in there